src/scripts/clipperUI/oneNoteApiUtils.ts

```diff
@@ -70,14 +118,16 @@ export module OneNoteApiUtils {
 		return responseCodeInfo ? responseCodeInfo.isRetryable : false;
 	}
 
+	/**
+	 * Retrieves response code information given that the context is in POSTing a clip.
```

Comment:
Not sure this is strictly true - doesn't sound like `30104/UserAccountSuspended`, `30105/OneDriveForBusinessNotProvisioned`, and maybe some others, will necessarily occur when posting a clip

In general, note that we use `oneNoteApiUtils.ts` to handle errors for more than just the `ClipToOneNoteAction` event